src/network/networkHandler.ts

```diff
@@ -682,6 +683,10 @@ async function handleOnDataMessage(d: OnDataArgs, overworld: Overworld): Promise
               // that they have, run the cinematic again.
               runCinematicLevelCamera(underworld);
             }
+            // Change end turn button from End Turn to Ready in multiplayer
+            const elEndTurnSpan = elEndTurnBtn.querySelector('[data-localize-text]') as HTMLElement;
+            elEndTurnSpan.dataset.localizeText = "ready"
```

Comment:
This line could probably be removed and we can directly just do elEndTurnSpan.innerText = i18n("ready") for example but I opted to keep it in to keep the HTML elements consistent

Reply:
localizeText automatically re-localizes the text when the games language is changed, so this is actually necessary in addition to setting the innerText